diesel/src/macros/mod.rs

```diff
@@ -23,10 +23,11 @@ macro_rules! __diesel_column {
             DB: $crate::backend::Backend,
             <$table as QuerySource>::FromClause: QueryFragment<DB>,
         {
-            fn walk_ast(&self, mut out: $crate::query_builder::AstPass<DB>) -> $crate::result::QueryResult<()> {
-                $table.from_clause().walk_ast(out.reborrow())?;
-                out.push_sql(".");
-                out.push_identifier($sql_name)
+            #[allow(non_snake_case)]
+            fn walk_ast(&self, mut diesel__out: $crate::query_builder::AstPass<DB>) -> $crate::result::QueryResult<()> {
```

Comment:
In procedural macros we tend to just prefix with `__`. Do you think it's worth being consistent with that convention here?

Reply:
To me, a leading underscore is an indication that the variable is not necessarily used, as it will disable "unused variable" warnings in the compiler.

Also, it seems plausible that people could have columns that start with an underscore. It seems less plausible that they will have columns starting `diesel__`.

Reply:
(To clarify: reserving this namespace is a breaking change because it will change people's existing schema, and reserving `diesel__` seems less likely to cause problems than reserving `__`)

Reply:
> Also, it seems plausible that people could have columns that start with an underscore.

Right, which is why we use double leading underscore

Reply:
I do not feel particularly strongly either way so have updated the PR.